jbpm-flow-builder/src/main/java/org/jbpm/process/workitem/WorkItemRepository.java

```diff
@@ -128,14 +137,35 @@
 			content = ConfFileUtils.URLContentsToString(
 				new URL(path + "/index.conf"));
 		} catch (Exception e) {
-			// Do nothing
+			// get directory listings if index.conf does not exist
+			return getDirectoriesNoIndexConfig(path);
 		}
 		if (content == null) {
 			return new String[0];
 		}
 		return content.split("\n");
 	}
 
+	private static String[] getDirectoriesNoIndexConfig(String path) {
+		List<String> dirs = new ArrayList<>();
+		try {
+			URL url = new URL(path);
+			BufferedReader reader = new BufferedReader(new InputStreamReader(url.openStream(), IoUtils.UTF8_CHARSET));
+			String line = null;
+
+			while((line = reader.readLine()) != null) {
+				if(!line.contains(".")) {
+					dirs.add(line);
+				}
+			}
+
+			reader.close();
+			return dirs.toArray(new String[0]);
```

Comment:
I believe this would be more effective as: `dirs.toArray(new String[dirs.size()]);`